src/svg2pdf.js

```diff
@@ -1226,7 +1222,9 @@ SOFTWARE.
     }
 
     if (markerEnd || markerStart || markerMid) {
-      lines.markers.draw((new Context(context)).set("transform"));
+      var markerContext = new Context(context);
+      markerContext.transform = _pdf.unitMatrix;      
+      lines.markers.draw(markerContext);
```

Comment:
How about this API:

```
  lines.markers.draw(context.nested({transform:_pdf.unitMatrix}))
```

make the nested context creation a method on the context and pass new properties as a property bag to the factory method. that looks more javascripty to me - and is a lot shorter, too!

Reply:
@HackbrettXXX @zenker981 ... - what do you think?

Reply:
I had already implemented something like this, but didn't push it before the merge - I'll try to reinclude it...